README.md

```diff
@@ -34,33 +34,40 @@ pyenv global system 3.8.10
 
 # Installation
 
-Create and activate a virtual environment.
-
-```shell
-python -m venv env
-source env/bin/activate
-```
-
-Installing volttron-platform-driver requires a running volttron instance.
-
-```shell
-pip install volttron
-
-# Start platform with output going to volttron.log
-volttron -vv -l volttron.log &
-```
-
-Install and start the volttron-platform-driver.
-
-```shell
-vctl install volttron-platform-driver --vip-identity platform.driver --start
-```
-
-View the status of the installed agent
-
-```shell
-vctl status
-```
+  1. Create and activate a virtual environment.
+
+     ```shell
+     python -m venv env
+     source env/bin/activate
+     ```
+
+  2. Installing volttron-platform-driver requires a running volttron instance.
+
+     ```shell
+     pip install volttron
+     ```    
+     Start platform with output going to volttron.log
+     ```shell
+     volttron -vv -l volttron.log &
+     ```
+
+  3. Install and start the volttron-platform-driver.
+
+     ```shell
+     vctl install volttron-platform-driver --vip-identity platform.driver --start
+     ```
+     #### Note:
+      In the above command, if no --vip-identity is not provided the default value would be "platform.driver". This comes  
```

Comment:
Just for my learning and to show why this change is necessary, this behavior of 'vctl install' is encoded in [this code block from aip.py](https://github.com/eclipse-volttron/volttron-core/blob/develop/src/volttron/server/aip.py#L530-L542), which was changed because of this [volttron-core PR](https://github.com/eclipse-volttron/volttron-core/pull/135/files).

We should probably document this `--vip-identity` behavior somewhere perhaps in copier-agent or readthedocs for modular Volttron. Even though I don't think we'll ever change this behavior of `vctl install`, this sounds like an interface that agents should know about if they want to have a default vip-identity.

Reply:
@schandrika is going to document it inside on confluence until we have a space out for documentation for modular.